chia/cmds/wallet_funcs.py

```diff
@@ -68,6 +100,35 @@ async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerpri
                 break
 
 
+async def get_csv(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    wallet_id = args["id"]
+    offset = args["offset"]
+    file_path = args["file"]
+    date_fmt = args["date_fmt"]
+
+    txs: List[TransactionRecord] = await wallet_client.get_csv(wallet_id)
+    config = load_config(DEFAULT_ROOT_PATH, "config.yaml", SERVICE_NAME)
+    name = config["network_overrides"]["config"][config["selected_network"]]["address_prefix"]
+    if len(txs) == 0:
```

Comment:
this is equivalent to just 
```python
if txs:
```

Reply:
Thanks for all your help!  The 2 suggestions above I understand and have made some updates to my code, but I'm not sure about this one?  How is this equivalent?

Reply:
No, the following is the case: 
empty list `[]` (as well as `None` and `0`) do the same thing for an `if` statement as `False`. 
So 
```python
if len(my_list) > 0:
    print("was nonempty")
else:
    print("was empty")
```
is equivalent to
```python
if len(my_list):
    print("was nonempty")
else:
    print("was empty")
```
and also to just
```python
if my_list:
    print("was nonempty")
else:
    print("was empty")
```
In the same vain, you can simplify `len(my_list) == 0` to doing a check against just `my_list` and putting the result in the else-clause.

(You can even do `not my_list`, which will evaluate to `False`. You can check it:
`not [3,4] == False`
but I'm not such a big fan of that since then `not` maps between different types)

But that was just for your info, imho you can leave this just as well.